static/app/components/u2f/u2finterface.tsx

```diff
@@ -116,11 +121,18 @@ class U2fInterface extends Component<Props, State> {
           () => {
             const u2fResponse = this.getU2FResponse(data);
             const challenge = JSON.stringify(this.props.challengeData);
+            const stateData = this.props.stateData
+              ? JSON.stringify(this.props.stateData)
+              : this.props.stateData;
```

Comment:
`this.props.stateData` is currently undefined, but when we later need to have a django view render an html template which renders this component, it will be set to an empty string